content/en/docs/tasks/access-application-cluster/communicate-containers-same-pod-shared-volume.md

```diff
@@ -7,7 +7,7 @@ weight: 110
 {{% capture overview %}}
 
 This page shows how to use a Volume to communicate between two Containers running
-in the same Pod.
+in the same Pod. See also how to allow processes to communicate by [sharing process namespace](/docs/tasks/configure-pod-container/share-process-namespace/) between containers (Kubernetes server must be at or later than version v1.10).
```

Comment:
Would it make sense to use a [feature state shortcode](https://kubernetes.io/docs/contribute/style/hugo-shortcodes/#feature-state) here, to highlight the need for Kubernetes v1.10 or later?

Reply:
(If using feature-states, it'll be important to show the reader what that applies to. The feature-state for sharing volumes is different from the feature state for the further reading about sharing processes between containers in a Pod).

Reply:
indeed, the page linked already clarifies feature state, adding it to the reference which is in the volumes documentation would be confusing.

Can you please re-clarify what changes you want me to make (if any) @sftim ?